server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -85,11 +84,9 @@ function sortedDayData(participantAssessmentData, participant_assessments) {
     ({ day }) =>
```

Comment:
Is it possible/realistic to do this sorting and filtering in the database query?

Reply:
Nevermind, we can optimize on another pass